cmd/setup-etcd-environment/run.go

```diff
@@ -86,9 +105,54 @@ func runRunCmd(cmd *cobra.Command, args []string) error {
 		out = f
 	}
 
-	if err := writeEnvironmentFile(map[string]string{
-		"DISCOVERY_SRV": runOpts.discoverySRV,
-	}, out, true); err != nil {
+	export := make(map[string]string)
+	etcdName := os.Getenv("ETCD_NAME")
+	if etcdName == "" {
+		return fmt.Errorf("environment variable ETCD_NAME has no value")
+	}
+	var e EtcdScaling
```

Comment:
could you use a more descriptive name than `e`, it's hard to remember what this stands for as i read thru below.

Reply:
sounds good.